docs/product-assembly_revising-projects/src/main/asciidoc/main.adoc

```diff
@@ -0,0 +1,35 @@
+[id='assembly_revising-projects']
+
+include::product-shared-docs/document-attributes.adoc[]
+
+= Revising projects in {CENTRAL}
+// Context attribute is assembly specific and enables module reuse between assemblies.
+:imagesdir: topics/product-shared-docs/images
+:context: revising-projects
+include::product-shared-docs/author-group.adoc[]
+
+// Purpose statement for the assembly
+As a process administrator, you can use {CENTRAL} in {PRODUCT} to import and revise existing projects.
+
+.Prerequisites
+* {EAP_LONG} 7.1.0 is installed. For installation information, see the https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.1/html-single/installation_guide/[_Red Hat JBoss EAP 7.1.0 Installation Guide_].
+* {PRODUCT} is installed and configured with {KIE_SERVER}. For more information see {URL_INSTALLING_ON_EAP}[_{INSTALLING_ON_EAP}_].
```

Comment:
missing comma in "For more information see".

Reply:
Fixed. Thank you!